### PR TITLE
added disjointness tutorial

### PR DESCRIPTION
**Pull request by shawntanzk**
taken directly from https://ontology101tutorial.readthedocs.io/en/latest/Disjointness.html

### CODE REVIEW

File: docs/tutorial/resources/BDK14-exercises/basic-dl-query/.svn/all-wcprops [hunk 1]
```diff
@@ -0,0 +1,17 @@
+K 25
```
Comment (matentzn) [inline docs/tutorial/resources/BDK14-exercises/basic-dl-query/.svn/all-wcprops:1]: why do we need svn files?

File: docs/tutorial/resources/BDK14-exercises/basic-dl-query/.svn/entries [hunk 2]
```diff
@@ -0,0 +1,96 @@
+10
```
Comment (matentzn) [inline docs/tutorial/resources/BDK14-exercises/basic-dl-query/.svn/entries:1]: delete .svn directory

File: docs/tutorial/resources/BDK14-exercises/basic-dl-query/catalog-v001.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (matentzn) [inline docs/tutorial/resources/BDK14-exercises/basic-dl-query/catalog-v001.xml:1]: why is this needed?

File: docs/tutorial/resources/BDK14-exercises/taxon-union/catalog-v001.xml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (matentzn) [inline docs/tutorial/resources/BDK14-exercises/taxon-union/catalog-v001.xml:1]: Why is this needed?

**Review by matentzn — CHANGES_REQUESTED** [review]
Why do we need a recommendedreadings.pdf - better add to markdown?

File: docs/tutorial/resources/BDK14-exercises/basic-dl-query/catalog-v001.xml (hunk 3 above)
Comment (shawntanzk) [reply docs/tutorial/resources/BDK14-exercises/basic-dl-query/catalog-v001.xml:1]: no idea, copied the whole folder across from the repo - will remove all catalog

**Review by matentzn — APPROVED** [review]
Didnt read the tutorial, but formally looks good!
